.buildkite/pipeline-resource-definitions/security-solution-quality-gate/kibana-serverless-security-solution-quality-gate-detection-engine.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-detection-engine
+  description: "[MKI] Executes Cypress tests for the Detection Engine team"
+spec:
+  type: buildkite-pipeline
+  owner: 'group:security-engineering-productivity'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-detection-engine
```

Comment:
Note: this name will be the displayed name in buildkite (that you see on the UI), and it's kebab-case slugified version will be the URL